games-misc/sdljoytest/sdljoytest-11102003.recipe

```diff
@@ -1,28 +1,49 @@
-DESCRIPTION="Simple Direct Media Layer"
+SUMMARY="Select all joysticks found on the system"
+DESCRIPTION="SDLJoytest is a program which will allow the user to select \
+from all joysticks found on the system (by SDL), and then test that selected \
+joystick or other game controller."
 HOMEPAGE="http://sdljoytest.sourceforge.net"
-SOURCE_URI="http://sourceforge.net/projects/sdljoytest/files/sdljoytest/SDLJoytest-GL/SDLJoytest-GL-11102003.tar.bz2/download"
-CHECKSUM_MD5="bd80f7a1537dc0e17c9d524f0b29f5ee"
+COPYRIGHT="2003 Samuel E. Bray"
+LICENSE="GNU GPL v2"
 REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
+SOURCE_URI="http://sourceforge.net/projects/sdljoytest/files/sdljoytest/SDLJoytest-GL/SDLJoytest-GL-11102003.tar.bz2"
```

Comment:
I think it could be nice to change it to:
```
SOURCE_URI="http://downloads.sourceforge.net/sdljoytest/SDLJoytest-GL-$portVersion.tar.bz2"
```
It's shorter and it saves a couple of HTTP redirects.
BTW, you can even make it slighly shorter with:
```
SOURCE_URI="http://downloads.sf.net/sdljoytest/SDLJoytest-GL-$portVersion.tar.bz2"
```
(I have no opinion, however,  on which is best)

Hmm, there's a trailing tab after the closing quote in the definition of BUILD_PREREQUIRES.